app/components/clock/Clock.jsx

```diff
@@ -11,19 +11,40 @@ class Clock extends React.Component {
     };
 
     state = {
-        dateNow: this._getCurrentDate()
+        hours:      this._getCurrentHours(),
+        minutes:    this._getCurrentMinutes(),
+        seconds:    this._getCurrentSeconds()
     };
 
     _updateDate = () => {
         this.setState({
-            dateNow: this._getCurrentDate()
+            hours:      this._getCurrentHours(),
+            minutes:    this._getCurrentMinutes(),
+            seconds:    this._getCurrentSeconds()
         });
     };
 
-    _getCurrentDate() {
+    _getCurrentHours() {
         const date = new Date();
+        const hours_24 = date.getHours();
 
-        return date.toLocaleTimeString(this.props.hourFormat === '12' ? 'en-US' : 'en-GB');
+        let hours_12 = hours_24 % 12;
+
+        hours_12 = hours_12 ? hours_12 : 12;
+
+        return(this.props.hourFormat === '12' ? hours_12 : hours_24);
```

Comment:
There should always be *space* symbol after `return` statement.
Also there is no need to use parenthesis in this case.